cssom-view-1/Overview.bs

```diff
@@ -1781,46 +1781,76 @@ Document includes GeometryUtils;
+	5. multiply the parentTransformationMatrix with the transformationMatrix and store in transformationMatrix
+	6. if currentElement is the anchestor or the anchestor is null return the transformationMatrix
+	</ol>
+</ol>
 
-  pseudo-elements before/after are children of the element
+The <dfn method for=GeometryUtils lt="getBoxQuads(options)|getBoxQuads()">getBoxQuads(<var>options</var>)</dfn> method must run the following steps:
 
-  viewport boxes are all the same
+<ol>
+1. create an array as result container
+2. for each css-fragment of the element run the following steps
+	<ol>
+	1. Create a DOMRect of the Element with the x,y value of 0 and the offsetWith and offsetHeight.
+	2. Depending on the value of options.box modify the Rect from 1. with the calculated style values of the element.
+	   - 'border': no change
+	   - 'margin': x - margin.left, y - margin.top, width + margin.left + margin.right and height + margin.top + margin.bottom
+	   - 'padding': x + border.leftWidth, y + border.topWidth, width - border.leftWidth - border.rightWidth and height - border.topWidth - border.bottomWidth
+	   - 'content': x + border.leftWidth + padding.left, y + border.topWidth + padding.top, width - border.leftWidth - padding.left - border.rightWidth - padding.rigth and height - border.topWidth - padding.top - border.bottomWidth - padding.bottom
+	3. create a DomQuad from the rect via DOMQuad.fromRect
+	4. Calculate the resulting transformation matrix between the element and options.realtiveTo (if relativeTo is unset use window.defaultView) via the algorithm defined in getCompleteTransform
+	5. transform every Point of the quad via the matrix calculated in 4. and add the result to the result container
+  	</ol>
+ 3. return the result container
+ </ol>
 </ol>
 
 The <dfn method for=GeometryUtils lt="convertQuadFromNode(quad, from, options)|convertQuadFromNode(quad, from)">convertQuadFromNode(<var>quad</var>, <var>from</var>, <var>options</var>)</dfn> method
 must run the following steps:
 
 <ol>
- <li><p class=issue>...
+ 1. transform each point of the DOMQuad via convertPointFromNode
 </ol>
 
 The <dfn method for=GeometryUtils lt="convertRectFromNode(rect, from, options)|convertRectFromNode(rect, from)">convertRectFromNode(<var>rect</var>, <var>from</var>, <var>options</var>)</dfn> method
 must run the following steps:
 
 <ol>
- <li><p class=issue>...
+ 1. create a DOMQuad from the rect via DOMQuad.fromRect
+ 2. transform each point of the DOMQuad from 1. via convertPointFromNode
 </ol>
 
 The <dfn method for=GeometryUtils lt="convertPointFromNode(point, from, options)|convertPointFromNode(point, from)">convertPointFromNode(<var>point</var>, <var>from</var>, <var>options</var>)</dfn>
 method must run the following steps:
 
 <ol>
- <li><p class=issue>...
+1. get the resulting matrix between the element and the document.defaultView via the algorithm defined in getCompleteTransform
```

Comment:
All these algorithms need to be in terms of `node`s, as `GeometryUtils` also applies to text nodes and such.

Reply:
Is node correct? Or should it be "GeometryNode"?
<img width="757" height="87" alt="image" src="https://github.com/user-attachments/assets/b7dd425e-8d78-4dce-955d-bd01b5c1b483" />